packages/kernel/src/tokens.ts

```diff
@@ -128,7 +128,7 @@ export interface IKernel extends IObservableDisposable {
    *
    * @param content - The content of the request.
```

Comment:
The docs need to be updated input_request -> input_reply

Reply:
Done!